### DISCUSSION TITLE
A few questions about changing entity ids

### DISCUSSION THREAD

**Discussion by juzi214032** [zachowj/node-red-contrib-home-assistant-websocket#1713] [Q&A]
1. When I changed the entity id in homeassistant, the entity lost its association with the device generated by node red, is this expected?
2. When I change the entity id in homeassistant, delete the integration and re-add it, at this point the entity id will be restored

**Comment by zachowj** [comment]
1. No.  
2. If you don’t delete the entity configuration in Node-RED, it will attempt to recreate the entity each time it connects to a Home Assistant instance with the integration installed.
